core/field_colour.js

```diff
@@ -138,11 +143,38 @@ Blockly.FieldColour.prototype.getText = function() {
 
 /**
  * An array of colour strings for the palette.
- * See bottom of this page for the default:
- * http://docs.closure-library.googlecode.com/git/closure_goog_ui_colorpicker.js.source.html
+ * Copied from goog.ui.ColorPicker.SIMPLE_GRID_COLORS
```

Comment:
Add: all colour pickers will use this set of colours unless overridden with setColours.

Reply:
Done.